components/Starknet/modules/starkgate/pages/overview.adoc

```diff
@@ -14,15 +14,15 @@ include::partial$snippet_backwards_compatibility_note.adoc[]
 [#starkgate_addresses]
 == StarkGate addresses
 
-L1 and L2 addresses for StarkGate bridges and supported tokens are listed in the JSON files in the Starknet GitHub repository shown in the table xref:#table_StarkGate_token_addresses[].
+L1 and L2 addresses for StarkGate bridges and supported tokens are listed in the JSON files shown in the table xref:#table_StarkGate_token_addresses[].
 
 [#table_StarkGate_token_addresses]
 .StarkGate bridged tokens and their addresses
 |===
 | Network | StarkGate bridged tokens JSON file
 
-| Mainnet | link:https://github.com/starknet-io/starknet-addresses/blob/master/bridged_tokens/mainnet.json[mainnet.json]
-| Sepolia testnet | link:https://github.com/starknet-io/starknet-addresses/blob/master/bridged_tokens/sepolia.json[sepolia.json]
+| Mainnet | link:https://beta.starkgate.starknet.io/static/tokens.json[mainnet.json]
```

Comment:
These files are no longer called "mainnet.json" and "sepolia.json" anymore, so please remove their names (i.e., the square bracket parts)
Please also change "Sepolia testnet" to "Sepolia".